app/pages/project/home.cjsx

```diff
@@ -33,6 +36,10 @@ module.exports = React.createClass
     undefined # Don't prevent default Link behavior.
 
   render: ->
+    logClick = @context.geordi?.makeHandler? 'get-started'
+    [owner, name] = @props.project.slug.split('/')
```

Comment:
Don't think this is necessary. The component is using `@props.project.slug` where necessary and I don't see `owner` or `name` being used anywhere anyway.